app/assets/javascripts/boot.coffee

```diff
@@ -11,6 +11,19 @@ drawMenuPlanRecipeTotal = ->
   totalRecipes = window.currentMenuPlan.get('recipeIds').length
   $('.recipe-count').text(totalRecipes)
 
+drawMenuPlanDropdown = ->
+  navDropdown = document.getElementById("menu_plan_dropdown")
+  ReactDOM.render React.createElement(MenuPlanDropdown, recipes: window.currentMenuPlanRecipes, deleteRecipe: removeRecipe), navDropdown
+
+removeRecipe = (recipe) ->
+  id = recipe.get('id')
+  window.currentMenuPlan.removeRecipeId(id)
+  window.currentMenuPlanRecipes.remove({id: id})
```

Comment:
Here’s a :necktie: suggestion:

You have `currentMenuPlan` and `currentMenuPlanRecipes` and do double-entry work like this to keep them both up-to-date in several places.

You can, instead, modify the `MenuPlanPresenter` to present `recipes`:

```diff
  def as_json(*)
    { id: menu_plan.id,
      name: menu_plan.name,
-     recipeIds: menu_plan.recipe_ids }
+     recipes: menu_plan.recipes.map { |recipe|
+     { id: recipe.id,
+       name: recipe.name,
```

You can have `menu_plans_controller.rb` render the menu plan again rather than `render json: {}`

and, finally, in `menu_plan.coffee`, you can do something like this:

```coffee
class @MenuPlan extends Backbone.Model
  # ...
  recipes: ->
    @_recipes ||= new Recipes(@get('recipes'))
```

Then, you can refer to `currentMenuPlan.recipes()`